randovania/games/factorio/game_data.py

```diff
@@ -74,12 +73,31 @@ def _hash_words() -> list[str]:
     presets=[
         {"path": "starter_preset.rdvpreset"},
     ],
-    faq=[],
+    faq=[
+        ("What versions of the game is supported?", "Only Factorio 2.0 is supported."),
+        ("Is Factorio: Space Age supported?", "No."),
+        ("Are other mods supported?", "Only mods that don't add new researches are compatible."),
+        (
+            "Is multiplayer supported?",
+            "Yes. Since the generated randomizer mod file isn't in the mod portal, "
+            "it'll be necessary to either share the mod file or have everyone export it themselves.",
+        ),
+    ],
+    web_info=randovania.game.web_info.GameWebInfo(
+        what_can_randomize=[
+            "The tech tree",
+            "The recipes for all Science Packs",
+            "The recipes for rocket parts",
+        ],
+        need_to_play=[
+            "A copy of the game from Steam or Factorio.com",
```

Comment:
`factorio.com` instead?
Also CC @duncathan for whether website allows markdown. I don't think so, but it may allow html?
based off of looking at it here: https://github.com/randovania/randovania.github.io/blob/418a7ab39ab408d1740ec35376426def44954d1a/_layouts/game.html#L42